internal/services/icinga2/docker.go

```diff
@@ -171,6 +171,20 @@ func (n *dockerInstance) WriteConfig(file string, data []byte) {
 	}
 }
 
+func (n *dockerInstance) DeleteConfigGlob(glob string) {
+	logger := n.logger.With(zap.String("glob", glob))
+
+	stderr := utils.NewLineWriter(func(line []byte) {
+		logger.Error("error from container while deleting file", zap.ByteString("line", line))
+	})
+
+	err := utils.DockerExec(context.Background(), n.icinga2Docker.dockerClient, n.logger, n.containerId,
+		[]string{"perl", "-e", "unlink glob @ARGV[0]", "--", "/" + glob}, nil, nil, stderr)
```

Comment:
Perl... ?
Perl?!

Reply:
I share that feeling but do you have better ideas? Should only use stuff already present in the `icinga/icinga2` containers and ideally doesn't have surprise features like `{"sh", "-c", "rm /"+glob"}` would have.

Note to self: if we keep Perl, `or die $!` should be added to have some error handling.

Reply:
surprise features?

Reply:
Whitespace, variables, backticks in names being interpreted by the shell.